### PR TITLE
Update CloudTrail - First draft of AWS integration edits

### PR DESCRIPTION
**Pull request by ellis-elastic**
## What does this PR do?

Added new content based on the new documentation guidelines.

From https://github.com/elastic/integrations/issues/3572:

>In https://github.com/elastic/integrations/pull/3308 we updated docs for two AWS integrations to align with the new documentation guidelines and establish the relationship between the AWS integration/package ("AWS") and integrations for individual AWS services (for example, "AWS CloudFront").
>
>Now we should update the docs for all AWS integrations for individual services to follow the same format as the updated "AWS CloudFront" integration docs.

This PR adds more context the AWS <service> integration including:

- [x] Adds context to the "Overview" including a link to the relevant AWS page and an example
- [x] Lists the types of "Data streams" for the service
- [x] "Requirements" points back to "AWS" for detailed information on credentials and permissions
- [x] "Requirements" includes any other service-specific requirements
- [x] "Setup" establishes a relationship between the AWS integration/package ("AWS") and this integration
- [x] Includes "Reference" sections

## Checklist

- [ ] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [ ] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [ ] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## Author's Checklist

- [ ] Review by docs team 
- [ ] Review by integrations team

## Related issues

- https://github.com/elastic/integrations/issues/3572

### CODE REVIEW

**Comment by ellis-elastic** [comment]
Does CloudTrail log file integrity validation need to be turned on? Is that a requirement?

File: packages/aws/_dev/build/docs/cloudtrail.md [hunk 1]
```diff
@@ -1,12 +1,51 @@
...
+If you want to collect data from two or more AWS services, consider using the **AWS** integration.
+
+When you configure the AWS integration, you can collect data from as many AWS services as you'd like.
+
+For step-by-step instructions on how to set up an integration, see the
+[Getting started](https://www.elastic.co/guide/en/welcome-to-elastic/current/getting-started-observability.html) guide.
+
+## Logs reference
 
 The `cloudtrail` dataset collects the AWS CloudTrail logs. CloudTrail monitors
```
Comment (colleenmcginnis) [inline packages/aws/_dev/build/docs/cloudtrail.md:42]:
```suggestion
The `cloudtrail` data stream collects the AWS CloudTrail logs. CloudTrail monitors 
```

**Review by colleenmcginnis — COMMENTED** [review]
Some minor suggestions below.

File: packages/aws/_dev/build/docs/cloudtrail.md [hunk 2]
```diff
@@ -1,12 +1,50 @@
...
-## Logs
+## Overview
+The AWS CloudTrail integration allows you to monitor [AWS CloudTrail](https://aws.amazon.com/cloudtrail/).
 
-The `cloudtrail` dataset collects the AWS CloudTrail logs. CloudTrail monitors 
-events for the account. If user creates a trail, it delivers those events as log
- files to a specific Amazon S3 bucket. The `cloudtrail` dataset does not read 
- the CloudTrail Digest files that are delivered to the S3 bucket when Log File 
- Integrity is turned on, it only reads the CloudTrail logs.
+Use the AWS CloudTrail integration to collect and parse logs related to account activity.
```
Comment (bmorelli25) [inline packages/aws/_dev/build/docs/cloudtrail.md:6]: What account(s) activity? What if we update this to:

```suggestion
Use the AWS CloudTrail integration to collect and parse logs related to account activity across your AWS infrastructure.
```

File: packages/aws/_dev/build/docs/cloudtrail.md [hunk 3]
```diff
@@ -1,12 +1,50 @@
...
+When you configure the AWS integration, you can collect data from as many AWS services as you'd like.
+
+For step-by-step instructions on how to set up an integration, see the
+[Getting started](https://www.elastic.co/guide/en/welcome-to-elastic/current/getting-started-observability.html) guide.
+
+## Logs reference
+
+The `cloudtrail` data stream collects the AWS CloudTrail logs. CloudTrail monitors 
+events for the account. If a user creates a trail, it delivers those events as log
+ files to a specific Amazon S3 bucket.
```
Comment (bmorelli25) [inline packages/aws/_dev/build/docs/cloudtrail.md:43]: I think we can add a little more description here. What about this?

```suggestion
The `cloudtrail` data stream collects AWS CloudTrail logs. CloudTrail monitors events like
user activity and API usage in AWS services. If a user creates a trail, it delivers those events as log
files to a specific Amazon S3 bucket. 
```

**Pull request merged**